### DISCUSSION TITLE
What is the PyTorch version that is most compatible with CUDA 12.4?

### DISCUSSION THREAD

**Discussion by BogeunPark** [MDIL-SNU/SevenNet#145] [General]
Hello, I am a first-year graduate student. In our lab, we had been using a GPU with CUDA 12.1, so we installed PyTorch version 2.2.2 for that.
However, we recently purchased a neuron from ksc, and I am installing SevenNet by myself, but the CUDA version is 12.4. I’m unsure which version of PyTorch I should install to ensure compatibility with CUDA 12.4.

**Comment by YutackPark** [comment]
I recommend 2.4.1

pip install torch==2.4.1 --index-url https://download.pytorch.org/whl/cu124

The versions on main page is not mandatory. They are just versions we tested or used. Other version combinations torch support have to work with SevenNet. If not, please report issue. Just do not use torch = 2.5.0. It has issue with lammps.

**Reply by BogeunPark** [reply]
Thank you for your kind response!
